src/main/scala/com/gu/newsletterlistcleanse/GetCleanseListLambda.scala

```diff
@@ -23,7 +24,8 @@ import scala.concurrent.{Await, Future}
 object GetCleanseListLambda {
 
   val logger: Logger = LoggerFactory.getLogger(this.getClass)
-  val campaigns: Campaigns = new CampaignsFromDB()
+  val serviceAccountCredentials: InputStream = this.getClass.getClassLoader().getResource("service-account.json").openStream()
+  val databaseOperations: DatabaseOperations = new BigQueryOperations(serviceAccountCredentials)
```

Comment:
Given that we're looking at currently implementing both Athena and Big Query access in parallel for a period of time, should the logic around `serviceAccountCredentials` be moved into the `BigQueryOperations` class? 

Potentially we pass the file name here to the constructor, but then have the code to open the stream as part of the `apply()` function for the class?

Reply:
I think I prefer the Operation to not be aware of how we're loading the credentials. I have a slight preference for that kind of detail to be owned by the wiring phase, which in this case is the construction of the lambda.